lib/services/slack.ts

```diff
@@ -0,0 +1,14 @@
+import { env } from '@/env.mjs';
+import { ofetch } from 'ofetch';
+
+export const notify = async (message: string) => {
+  try {
+    await ofetch(env.SLACK_POSTMAN_WEBHOOK_URL, {
+      body: { text: message },
+      method: 'POST',
+    });
+  } catch (error) {
+    // Do nothing?
```

Comment:
yeah 😃

Reply:
guess we could log to `baselime` or `sentry` when we have em